rest/admin_api_test.go

```diff
@@ -2013,3 +2013,120 @@ func TestSessionExpirationDateTimeFormat(t *testing.T) {
 	assert.NoError(t, err, "Couldn't parse session expiration datetime")
 	assert.True(t, expires.Sub(time.Now()).Hours() <= 24, "Couldn't validate session expiration")
 }
+
+func TestUserAndRoleResponseContentType(t *testing.T) {
+	rt := NewRestTester(t, nil)
+	defer rt.Close()
+
+	// Create a user 'alice' through PUT request.
+	body := `{"email":"alice@couchbase.com","password":"cGFzc3dvcmQ=","admin_channels":["foo", "bar"]}`
+	response := rt.SendAdminRequest(http.MethodPut, "/db/_user/alice", body)
+	assert.Equal(t, http.StatusCreated, response.Code)
+	assert.Equal(t, "application/json", response.Header().Get("Content-Type"))
```

Comment:
Might be worth asserting no error when JSON unmarshalling the response body into a `map[string]interface{}` so we know it's valid JSON when the content type header is set.

Reply:
Yeah, it's worth asserting no error when JSON unmarshalling the response body into a map[string]interface{} so we know it's valid JSON when the content type header is set. Fixed in latest revision.